src/DefaultMappingStrategy.ts

```diff
@@ -0,0 +1,21 @@
+import { ProblemDocument } from 'http-problem-details'
+import { IMappingStrategy } from './IMappingStrategy'
+import { MapperRegistry } from './MapperRegistry'
+
+export class DefaultMappingStrategy implements IMappingStrategy {
+  public registry: MapperRegistry
```

Comment:
Slightly unrelated, maybe make readonly
```suggestion
  public readonly registry: MapperRegistry
```